packages/freshmint/generate/index.ts

```diff
@@ -99,6 +99,15 @@ async function generateStandardProject(dir: string, contract: ContractConfig, in
 
   await writeFile(path.resolve(dir, 'cadence/transactions/mint_with_claim_key.cdc'), mintWithClaimKeyTransaction);
 
+  const destroyNFTsTransaction = CommonNFTGenerator.destroyNFTs({
+    imports: shiftedImports,
+    contractName: contract.name,
+    contractAddress,
+  });
+
+  await writeFile(path.resolve(dir, 'cadence/transactions/destroy_nfts.cdc'), destroyNFTsTransaction);
```

Comment:
I forgot to mention this before, but you'll need to add this code to the `generateEditionProject` function below as well.

I've been meaning to refactor this file to remove the duplicate code, but I think that can happen in a separate PR.